internal/util-complete/src/main/scala/sbt/internal/util/complete/Parser.scala

```diff
@@ -18,7 +18,7 @@ import sbt.internal.util.Util.{ makeList, separate }
  * The methods in this trait are what must be implemented to define a new Parser implementation, but are not typically useful for common usage.
  * Instead, most useful methods for combining smaller parsers into larger parsers are implicitly added by the [[RichParser]] type.
  */
-sealed trait Parser[+T] {
+trait Parser[+T] {
```

Comment:
I was curious how long back the `sealed` goes to, and it went back all the way to the beginning.
https://github.com/sbt/sbt-zero-thirteen/commit/8e6b42180d7ab45a3f31fe44f4d73dcbf59a4186